extensions/web-tracker/javascript/src/analytics.js-integration-apache-unomi.js

```diff
@@ -19,23 +19,26 @@
 var integration = require('@segment/analytics.js-integration');
 var extend  = require('extend');
 
-var Unomi = module.exports = integration('Apache Unomi')
+var Unomi = (module.exports = integration('Apache Unomi')
+    .global('cxs')
     .assumesPageview()
     .readyOnLoad()
-    .global('cxs')
     .option('scope', 'systemscope')
     .option('url', 'http://localhost:8181')
     .option('timeoutInMilliseconds', 1500)
     .option('sessionCookieName', 'unomiSessionId')
-    .option('sessionId');
+    .option('sessionId'));
 
 /**
  * Initialize.
  *
  * @api public
  */
-Unomi.prototype.initialize = function(page) {
+Unomi.prototype.initialize = function() {
     var self = this;
+
+    console.info('[UNOMI] Initializing...', arguments);
```

Comment:
where is the **`arguments`** variable declared?

Reply:
This is a standard built-in variables exposed by functions, see : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

Reply:
Yes, but the initialize function doesn't have any parameters at the moment

Reply:
Javascript allows passing arguments to functions that don't declare them. This log is here to check what the library is sending, as it has changed over time and is expected to change again.